src/main/java/com/kodilla/ecommercee/CartController/GlobalHttpErrorHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.kodilla.ecommercee.CartController;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class GlobalHttpErrorHandler extends ResponseEntityExceptionHandler {
```

Comment:
Im not sure if we should add Exceptions at this moment(in this task)

Reply:
As you wish.